client/cli/Cargo.toml

```diff
@@ -24,13 +24,23 @@ lazy_static = "1.4.0"
 app_dirs = "1.2.1"
 tokio = { version = "0.2.9", features = [ "signal", "rt-core", "rt-threaded" ] }
 futures = "0.3.4"
+futures01 = { package = "futures", version = "0.1.29" }
```

Comment:
I assume this was added because the original command had it, but do we _really_ need a 0.1-Future here? And if so, shouldn't we at least use the 0.3 compat-feature-future instead?